src/views/StackView/StackViewLayout.js

```diff
@@ -450,7 +459,19 @@ class StackViewLayout extends React.Component {
     const headerMode = this._getHeaderMode();
     let marginTop = 0;
     if (!hasHeader && headerMode === 'float') {
-      marginTop = -Header.HEIGHT;
+      const { isLandscape } = this.props;
+      let headerHeight;
+      if (Platform.OS === 'android') {
+        // TODO: Need to handle translucent status bar.
+        headerHeight = 56;
+      } else if (isLandscape && !Platform.isPad) {
```

Comment:
Hello :) I'm migrating to v2. Was this line supposed to target only iOS? If so we might need to add a `Platform.OS === ios`. On the web it looks like this:
![image](https://user-images.githubusercontent.com/82368/39893204-5387d78e-54a3-11e8-93d1-d82583935ba1.png)